pkg/scheduler/framework/session_plugins.go

```diff
@@ -304,6 +309,50 @@ func (ssn *Session) JobOrderFn(l, r interface{}) bool {
 
 }
 
+// NamespaceOrderEnabled returns the NamespaceOrder is enabled in this session or not
+// judging by length of namespaceOrderFns
+func (ssn *Session) NamespaceOrderEnabled() bool {
+	return len(ssn.namespaceOrderFns) > 0
+}
+
+// NamespaceOrderEnabledInPlugin the NamespaceOrder for this plugin is enabled in this session or not
+func (ssn *Session) NamespaceOrderEnabledInPlugin(pluginName string) bool {
+	for _, tier := range ssn.Tiers {
+		for _, plugin := range tier.Plugins {
+			if plugin.Name != pluginName {
+				continue
+			}
+			return isEnabled(plugin.EnabledNamespaceOrder)
+		}
+	}
+	return false
+}
+
+// NamespaceOrderFn invoke namespaceorder function of the plugins
+func (ssn *Session) NamespaceOrderFn(l, r interface{}) bool {
```

Comment:
So the order of namespace is decided by the first plugin if it has enabled the namespace order?

Reply:
Yes, if some plugin decide namespace A is prior to B, this decision is just the final result.

This logic follows the code pattern in other `CompareFn`, like `JobOrderFn`.